### DISCUSSION TITLE
Problem with swagger and returns of integers of length of 19 characters

### DISCUSSION THREAD

**Discussion by jbaumgarten** [fastapi/fastapi#9672] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from uvicorn import run

app = FastAPI()

@app.get("/test")
def read_test():
    return 1091339850071937124


if __name__ == "__main__":
    run("test:app", host="0.0.0.0", port=8000, log_level="debug", reload=True)
```

### Description

Open the browser and call the endpoint `/docs#/default/read_test_test_get`
It returns an integer with `1091339850071937200`
But I expected it to return `1091339850071937124`

If I call directly the endpoint `test`, the right value is returned

### Operating System

Linux

### FastAPI Version

0.95.1

### Python Version

Python 3.10.11

**Comment by diegolatorre116** [comment, marked as answer]
This is a javascript constraint that is used by Swagger UI to display API documentation. You can read more about it at: https://github.com/swagger-api/swagger-ui/issues/2030.
In short, unfortunately there is no workaround for this limitation at the moment. So, if you want it to appear correctly in swagger you will have to send it as a string, either directly in the return:

```
@app.get("/test").
def read_test():
    return "109133939850071937124"
``` 

Or using the response_model:

```
@app.get("/test", response_model=str)
def read_test():
    return 109133939850071937124
```
I hope you find the answer useful,

**Reply by jbaumgarten** [reply]
I simplified my code so it's not simply an int but I got what's the problem :D
Thx for the reply and for your time :)
